contracts/registry/SimpleRegistry.sol

```diff
@@ -100,7 +93,6 @@ contract SimpleRegistry is Ownable, MetadataRegistry, OwnerRegistry, ReverseRegi
         external
         payable
```

Comment:
If there is no fee payable can be removed

Reply:
true dat